src/components/generic/InputField/InputField.jsx

```diff
@@ -153,6 +161,8 @@ InputField.defaultProps = {
   value: '',
   name: '',
   placeholder: '',
+  maxLength: undefined,
+  pattern: '',
```

Comment:
`undefined` would be slightly better (instead of asking the input to match an empty string regex (which matches anything, actually, so it would work), it's a bit better to don't ask it at all to match anything)